plugins/plugin-git/che-plugin-git-ext-git/src/main/resources/org/eclipse/che/ide/ext/git/client/panel/gitPanel.css

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) 2012-2017 Red Hat, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+
+.git-panel-full-height {
+  height: 100%;
+}
+
+.git-panel-panel-top-indent {
+  margin-top: 5px;
+}
+
+.git-panel-panel-bottom-indent {
+  margin-bottom: 5px;
+}
+
+.git-panel-vertical-items {
+  width: 100%;
+  height: 100%;
+}
+
+.git-panel-vertical-items > tbody > tr:first-child {
+  height: 20px;
+}
+
+.git-panel-repository-changes-label {
+  float: right;
+  color: #252a38;
```

Comment:
Move the color definition to the theme interface.

Reply:
ok